CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+All contributors and participants agree to abide by its terms. To report
+violations, shoot out an email to founders@gitcoin.co
+
+The Code of Conduct is designed and intended, above all else, to help establish
+a culture within the project that allows anyone and everyone who wants to
+contribute to feel safe doing so.
+
+Open, diverse, and inclusive communities live and die on the basis of trust.
+Contributors can disagree with one another so long as they are done in good
+faith and everyone is working towards a common goal.
+
+## Issues
+Issues in `gitcoin/web` are the primary means by which bug reports and
+general discussions are made. An contributor is allowed to create an issue,
+discuss and provide a fix if needed.
+
+## Discussions And General Help
+As Gitcoin is still at it's early stages, drop by [gitcoin.co/slack](gitcoin.co/slack)
+and say hi to know what's next / to get your answers
+cleared up.
+
+## Pull Requests
+Pull Requests are the way in which concrete changes are made to the code and
+documentation.
+
+### Step 1: Fork
+
+Fork the project [on GitHub](https://github.com/gitcoinco/web) and clone your
+fork locally.
+
+```text
+$ git clone git@github.com:username/web.git
+$ cd web
+$ git remote add upstream https://github.com/gitcoinco/web.git
+$ git fetch upstream
+```
+
+### Step 2: Branch
+
+It's always better to create local branches to work on a specific issue. Makes
+life easier for you if you are the kind who enjoys multiple things parallely.
+These should also be created directly off of the `master` branch.
+
+```text
+$ git checkout -b my-branch -t upstream/master
+```
+
+### Step 3: Code
+
+As of now, we don't have any sort of design style / lint to validate things.
+So we ask you to ensure all these are met before you shoot out a PR.
+- Avoid trailing whitespace & un-necessary white lines
+- Indentation is as follows
+  - 1 tab = 2 spaces for `.html` files
+  - 1 tab = 4 spaces for everything else
+
+### Step 4: Commit
+
+1. The first line should:
+   - contain a short description of the change (preferably 50 characters or less
+     and no more than 72 characters)
+   - be prefixed with the name of the changed subsystem and start with an
+   imperative verb.
+   - If your commit can be explained in one line, then you can skip whats below
+
+   __Examples:__
+   - `doc: updated README.md`
+   - `ui: added design for testimonial page`
+
+2. Keep the second line blank.
+3. Wrap all other lines at 72 columns.
+4. If your PR fixed an issue, Use the `Fixes:` prefix and the full issue URL.
+  For other references use `Refs:`.
+
+   __Examples:__
+   - `Fixes: https://github.com/gitcoinco/web/issues/87`
+   - `Refs: https://github.com/gitcoinco/web/issues/91`
+
+5. Sample commit
+   ```txt
+   subsystem: explain the commit in one line
+
+   - list out your changes as points if there are many changes
```

Comment:
If there are many changes, they should probably be split off into separate PRs.

Reply:
Ah but what if they all correspond to the same issue ? doesn't that make reviewing messier ?
If they can be separated into 2 PRs , by all means we would allow it ^_^ 
I can update it there saying or shoot out multiple PRs if that seems feasible. Would that make things clearer ?